peeringdb_server/templates/site/register.html

```diff
@@ -78,7 +78,7 @@ <h3>{% trans "Create account" %}</h3>
              data-edit-name="last_name"
              data-edit-type="string" />
 
-      <div class="captcha">
+      <div class={% trans "captcha" %}>
```

Comment:
Using trans on CSS classes would break the look and feel

Reply:
That's finally removed!